route-rs-runtime/src/pipeline/mod.rs

```diff
@@ -1,2 +1,8 @@
+//! # What are they?
+//!
+//! Pipelines are abstractions used by graphgen to define a grouping of links that are run by the runtime, in our case Tokio. The graphgen
```

Comment:
"grouping" sounds like just a set of links. Maybe we can use "graph" to imply that edges are established between them?

Reply:
I'm not really sure what to do about pipelines right now. Ideally we won;t need them once we finish the router. The Link abstraction feels powerful enough, it can hook right to Afpacket or a PCAP input link.